### PR TITLE
Implement a Infinite Scroll on the home page with AJAX | Closes #49

### LINKED ISSUE #49: Implement Ajax
## Implement Ajax
We want to be able to save and fetch content from our backend.

**Branch: ```49-ajax ```**

### Tasks **(D.O.D)**
-  **Home Page**
    - [ ] Infinate scrol
    - [x] Fetch Post Titles from DB
    - [ ] Link Posts to correct Page
- **Post Page**
    - [ ] Fetch correct **Post data**
    - [ ] Show concent with the correct styling **(Markdown)**
    - [ ] Save data to the DB 
- **Profile Page**
    - [ ] Fetch correct **User data**

### PR DESCRIPTION
**Pull request by AjobK**
Closes #49 

**Updates**
Me and Miguel made the infinite scroll on the homepage

**Things to do for next sprint**
- Make sure that posts don't load constantly when stuck at the bottom of page
- When you scroll, log in and log out, and go to 'My Profile' you get an error

### CODE REVIEW

**Review by Samvdg — APPROVED** [review]

File: src/components/loader/loader.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+@keyframes load {
+	0% {
+		transform: rotate(0);
+		-moz-box-shadow: 0 0 0 0 rgba(255,64,64, 0.5);
+		box-shadow: 0 0 0 0 rgba(255,64,64, 0.5);
+	}
+	70% {
+		-moz-box-shadow: 0 0 0 10px rgba(255,64,64, 0);
+		box-shadow: 0 0 0 10px rgba(255,64,64, 0);
```
Comment (Samvdg) [inline src/components/loader/loader.scss:11]: can't you just do `box-shadow-left` ?
edit: (this ofcourse applies to all instances within the css)

File: src/components/posts/index.js [hunk 2]
```diff
@@ -27,6 +28,15 @@ class Posts extends Component {
       })
   }
 
+  // Checks if user is at end of page
+  isAtEndOfPage() {
```
Comment (Samvdg) [inline src/components/posts/index.js]: just questioning if the function name isn't already explained enough? unnecessary comment?

File: src/components/registerPrompt/index.js [hunk 3]
```diff
@@ -30,10 +30,10 @@ class RegisterPrompt extends Component {
...
           <div className={styles.image} />
         </div>
-        <p className={styles.textFooter}><small className={styles.textFooterSmall}>By registering I confirm that I have read and agree to the </small><a className={styles.textFooter_link}href='#'>Terms of service</a></p>
+        <p className={styles.textFooter}><small>By registering I confirm that I have read and agree to the </small><a className={styles.textFooter_link}href='#'>Terms of service</a></p>
       </div>
     )
   }
 }
 
-export default RegisterPrompt
```
Comment (MiguelFieira) [inline src/components/registerPrompt/index.js:39]: endline

**Review by MiguelFieira — CHANGES_REQUESTED** [review]
Linting

**Pull request closed without merging**